WordPress/src/androidTest/java/org/wordpress/android/InitializationRule.kt

```diff
@@ -23,17 +23,27 @@ class InitializationRule : TestRule {
     override fun apply(base: Statement?, description: Description?): Statement {
         return object : Statement() {
             override fun evaluate() {
-                instrumentation.runOnMainSync {
-                    val application = instrumentation.targetContext.applicationContext as Application
-                    val appInitializer = EntryPoints.get(
-                            application,
-                            AppInitializerEntryPoint::class.java
-                    ).inject()
+                if (!isInitialized) {
+                    instrumentation.runOnMainSync {
+                        val application = instrumentation.targetContext.applicationContext as Application
+                        val appInitializer = EntryPoints.get(
+                                application,
+                                AppInitializerEntryPoint::class.java
+                        ).initializer()
 
-                    appInitializer.init(application)
+                        appInitializer.init()
+                    }
+                    isInitialized = true
                 }
                 base?.evaluate()
             }
         }
     }
+
+    companion object {
+        // InitializationRule must be initialized only once. Otherwise some static functions in AppInitializer throws
+        // exception, if they are initialized more than once. (e.g. WebView.setDataDirectorySuffix(),
+        // EventBusBuilder.installDefaultEventBus)
+        private var isInitialized = false
```

Comment:
With `@Rule` annotation, the TestRule runs for each test cases. That means `InitializationRule` will be called for each test cases. But that causes crash because some functions in `AppInitializer` must be called only once for application lifecycle. 

With `isInitialized`, I ensured that `AppInitializer` will be called once. The disadvantage of this way can be that same `AppInitializer` instance will be used for each test cases. If I encounter this problem, I'll try to clean resources in `AppInitializer` before test cases.

I tried to use `@ClassRule` to run test rule once for this class, but it didn't work because of Hilt's Entry Point. I couldn't find the reason. I may try to use `@ClassRule` again later.